project-base/config/packages/snc_redis.yaml

```diff
@@ -29,12 +29,12 @@ snc_redis:
             alias: 'global'
             dsn: 'redis://%redis_host%'
             options:
-                prefix: '%env(REDIS_PREFIX)%'
+                prefix: '%env(REDIS_PREFIX)%%build-version%'
         session:
             type: 'phpredis'
             alias: 'session'
             dsn: 'redis://%redis_host%'
     session:
         client: 'session'
         ttl: 604800
-        prefix: '%env(REDIS_PREFIX)%session:'
+        prefix: '%env(REDIS_PREFIX)%%build-version%session:'
```

Comment:
I'm not sure, but this log out all users after deploy, or?

Reply:
You are right